### PR TITLE
Replacement of the Windows DHCP sensor section (UDP-Reflector) with DHCP-Forwarder.

### PR DESCRIPTION
**Pull request by tlaurion**
# Description
Replaces udp-reflector documentation and launches DHCP-Forwarder.

# Delete branch after merge
YES

# NEWS file entries
* DHCP-Forwarder Installer replaces complicated udp-reflector deployement. IP helpers no more?

## Enhancements
* We could receive only 18% of the DHCP traffic we currently receive, thanks to DHCP-Forwarder. (see dhcp-forwarder/TESTING.md)

# UPGRADE file entries
* Should we create a uninstall guide for udp-reflector?

### CODE REVIEW

**Comment by tlaurion** [comment]
Ready to merge?

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -3747,45 +3747,15 @@ These sensors work by capturing the packets at the lowest level possible on your
...
-* In 'Startup type' select Automatic
-
-In Log on:
-[options="compact"]
-* In 'Log on as' select Local System account
-
-Then press 'Install service'
-
-The DHCP traffic should now be reflected on your PacketFence server.
+The projet page is https://github.com/inverse-inc/packetfence-dhcp-forwarder[here].
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: *project

Comment (julsemaan) [reply docs/PacketFence_Administration_Guide.asciidoc]: Maybe better reworded in:
The project page can be found here

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -3747,45 +3747,15 @@ These sensors work by capturing the packets at the lowest level possible on your
 
 Microsoft DHCP sensor
 ^^^^^^^^^^^^^^^^^^^^^
+DHCP-Forwarder is an optimized version of precedent udp-reflector, which installs easily and only copy DHCPREQUESTS and DHCPACK packets to the destination. 
 
-You will first need to download and install 'WinPcap' available from http://www.winpcap.org/install/
+https://inverse.ca/downloads/PacketFence/windows-dhcp-forwarder/DHCP%20Forwarder%20Installer.exe[Download the installer here.]
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc:3752]: I'm being picky now so feel free to ignore my comment, but a download link with spaces in it is not really in my personal preferences

**Pull request merged**